pkg/security/securitycontextconstraints/selinux/mustrunas.go

```diff
@@ -2,9 +2,12 @@ package selinux
 
 import (
 	"fmt"
+	"sort"
+	"strings"
 
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	api "k8s.io/kubernetes/pkg/apis/core"
+	psputil "k8s.io/kubernetes/pkg/security/podsecuritypolicy/util"
```

Comment:
I decided to use PSP's utils instead of duplicating them. That's why this PR has `vendor-update` tag.